apps/participants/views.py

```diff
@@ -456,6 +458,35 @@ def get_participant_team_details_for_challenge(request, challenge_pk):
         return Response(response_data, status=status.HTTP_404_NOT_FOUND)
 
 
+@swagger_auto_schema(
+    methods=["post"],
+    manual_parameters=[
+        openapi.Parameter(
+            name="challenge_pk",
+            in_=openapi.IN_PATH,
+            type=openapi.TYPE_NUMBER,
+            description="Challenge Primary Key",
```

Comment:
change `Challenge Primary Key` to `Challenge id`

Reply:
Done

Reply:
@afif-fahim this is not updated. Did you push the commit?

Reply:
Sorry I have not! I am updating the PR in a moment.